.gitlab-ci.yml

```diff
@@ -151,21 +151,24 @@ variables:
   # To use images from datadog-agent-buildimages dev branches, set the corresponding
   # SUFFIX variable to _test_only
   DATADOG_AGENT_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_BUILDIMAGES: v20878799-a2f77ae
+  DATADOG_AGENT_BUILDIMAGES: v21999990-f991d55
   DATADOG_AGENT_WINBUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_WINBUILDIMAGES: v20878799-a2f77ae
+  DATADOG_AGENT_WINBUILDIMAGES: v21999990-f991d55
   DATADOG_AGENT_ARMBUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_ARMBUILDIMAGES: v20878799-a2f77ae
+  DATADOG_AGENT_ARMBUILDIMAGES: v21999990-f991d55
   DATADOG_AGENT_SYSPROBE_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_SYSPROBE_BUILDIMAGES: v20878799-a2f77ae
+  DATADOG_AGENT_SYSPROBE_BUILDIMAGES: v21999990-f991d55
   DATADOG_AGENT_KERNEL_MATRIX_TESTING_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_KERNEL_MATRIX_TESTING_BUILDIMAGES: v20878799-a2f77ae
+  DATADOG_AGENT_KERNEL_MATRIX_TESTING_BUILDIMAGES: v21999990-f991d55
   DATADOG_AGENT_NIKOS_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_NIKOS_BUILDIMAGES: v20878799-a2f77ae
+  DATADOG_AGENT_NIKOS_BUILDIMAGES: v21999990-f991d55
   DATADOG_AGENT_BTF_GEN_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_BTF_GEN_BUILDIMAGES: v20878799-a2f77ae
-  TEST_INFRA_DEFINITIONS_BUILDIMAGES: cb9e86993fd5
-  DATADOG_AGENT_BUILDERS: v19474930-fe4bcd9
+  DATADOG_AGENT_BTF_GEN_BUILDIMAGES: v21999990-f991d55
+  # To use images from test-infra-definitions dev branches, set the SUFFIX variable to -dev
+  # and check the job creating the image to make sure you have the right SHA prefix
+  TEST_INFRA_DEFINITIONS_BUILDIMAGES_SUFFIX: ""
+  TEST_INFRA_DEFINITIONS_BUILDIMAGES: d2510f65b6fe
+  DATADOG_AGENT_BUILDERS: v21429255-6b46caa
```

Comment:
These changes come from different PRs, I'm not sure we want to bring them to 7.49.x (ie. changing `TEST_INFRA_DEFINITIONS_BUILDIMAGES` and `DATADOG_AGENT_BUILDERS`) ?

Reply:
Good catch, these were from a mistaken cherry-pick. Fixed them.